### ISSUE TITLE
Add a camera AR mode

### ISSUE THREAD

**Issue by jaydeetay** [sky-map-team/stardroid#463]
Blocked on #460

**Comment by xandro0777** [comment]
Auto calibration would be great but we would have to live with realistically very limited value.

Do not wait for it.

**Comment by jaydeetay** [comment]
Status update: this is actually built already. Through-camera (AR) mode — camera passthrough, drag-to-align, and share — is fully implemented in Sky Map v2, gated behind an experiment flag (`camera_ar_enabled`) and currently shipping off by default while it's staged for rollout. Leaving this open until it's actually switched on for everyone; will close it out once it ships. Closed #483, #140, and #596 as duplicates of this one — same ask.
